src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -246,6 +246,22 @@ def _validate_service_tag_existence(cmd, namespace):
                                      'Cannot add duplicate Service Tag values.')
 
 
+def validate_staticsite_does_not_exist(cmd, namespace):
+    from azure.mgmt.web import WebSiteManagementClient
+    from azure.core.exceptions import ResourceNotFoundError
+
+    client = get_mgmt_service_client(cmd.cli_ctx, WebSiteManagementClient).static_sites
+    try:
+        client.get_static_site(namespace.resource_group_name, namespace.name)
+        exists = True
+    except ResourceNotFoundError:
+        exists = False
+
+    if exists:
+        raise ValidationError("Static Web App {} already exists in resource group {}".format(
```

Comment:
cli create command needs to be idempotent, infact the Create_or_Update APIs should be Idempotent, while I am not sure if this is a regression in the API, while we wait, lets keep this (hopefully) as a temporary fix & ensure that if the app exists we do a GET on the SWA & return that response. Thanks!